roles/baseline/defaults/main.yml

```diff
@@ -39,8 +39,8 @@ ingressVersions:
   k8sMinorVersionFloor:
     value: 24
     api:
-      chartVersion: 4.6.0
-      appVersion: 1.7.0
+      chartVersion: 4.7.1
+      appVersion: 1.8.1
```

Comment:
Still not sure why we include appVersion along with the chart version. This is deceptive in that folks that are reading the code could make the assumption this can be changed and unless we test every variation of the appVersion with a given chart being used this is not the case. Nothing to do with this feature, just believe we need to remove the appVersion in future releases.